FFMpegCore/FFMpeg/Arguments/MetaDataArgument.cs

```diff
@@ -15,13 +19,20 @@ public MetaDataArgument(string metaDataContent)
             _metaDataContent = metaDataContent;
         }
 
-        public string Text => $"-i \"{_tempFileName}\" -map_metadata 1";
+        public string Text => GetText(null);
 
         public Task During(CancellationToken cancellationToken = default) => Task.CompletedTask;
 
 
         public void Pre() => File.WriteAllText(_tempFileName, _metaDataContent);
 
         public void Post() => File.Delete(_tempFileName);
+
+        public string GetText(StringBuilder context)
```

Comment:
Just use `string` instead of `StringBuilder` here. Fits together with my comment below